contracts/SpotV1.sol

```diff
@@ -102,14 +90,18 @@ contract SpotV1 is ERC20Spot {
 
         // buy liquidity and stake
         if (msg.value > 0) {
+            require(swap0.path[0] == wrapper, "wrong path");
             amount = msg.value;
```

Comment:
WARNING: consider replacing with require(amount==msg.value, ...)
to avoid arguments misusing